pkgs/development/compilers/rust/1_39_0.nix

```diff
@@ -1,3 +1,8 @@
+# After updating check ...
+# 1. if rustc can produce rust binaries on x86_64-linux, aarch64-linux and x86_64-darwin:
+#    i.e. nix build -f . fd or @GrahamcOfBorg build fd on github
```

Comment:
What does `fd` mean? And to what directory does `-f .` refer?

Reply:
`fd` is just a simple rust example that should build on all platforms.